setup.py

```diff
@@ -273,6 +273,7 @@ def run(self):
     extras["flax"] = deps_list("jax", "jaxlib", "flax", "optax")
 
 extras["tokenizers"] = deps_list("tokenizers")
+extras["safetensors"] = deps_list("safetensors")
```

Comment:
```suggestion
extras["safetensors"] = deps_list("safetensors")
```
Like the one for tokenizers, this is really superfluous as you can't escape the packages in `install_requires`.